go.mod

```diff
@@ -30,6 +31,7 @@ require (
 	github.com/spf13/cobra v1.0.1-0.20200909172742-8a63648dd905
 	github.com/spf13/viper v1.7.0
 	github.com/stretchr/testify v1.6.0
+	github.com/vbatts/tar-split v0.11.1 // indirect
```

Comment:
I don't see this being used anywhere, was this intentional?

Reply:
This was _not_ intentional! Good catch!